kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/IRestClient.java

```diff
@@ -18,6 +18,7 @@
 package org.apache.kyuubi.client;
 
 import java.util.Map;
+import org.apache.kyuubi.client.api.v1.dto.MultiPart;
```

Comment:
nit: one more blank line

Reply:
It seems that the `dev/reformat` tool removes the blank line.